manifests/registry.pp

```diff
@@ -40,6 +40,7 @@
   Optional[String] $pass_hash                          = undef,
   Optional[String] $email                              = undef,
   Optional[String] $local_user                         = 'root',
+  Optional[String] $local_user_home                    = undef,
```

Comment:
The inclusion of a new parameter leads me to think that there should be tests in the spec/defines/registry_spec.rb to verify the behavior when this parameter is/is not called, as with username in the linked example;

https://github.com/puppetlabs/puppetlabs-docker/blob/master/spec/defines/registry_spec.rb#L28-L38